translation.py

```diff
@@ -343,6 +343,14 @@ def translateSimple(toTranslate, commands):
     proc_in.stdout.close()
     return translated.decode('utf-8')
 
+@gen.coroutine
+def translateNoFlush(toTranslate, commands):
```

Comment:
@sushain97 , the `assert proc_in == proc_out` line in `translateSimple` which is not present in `translateNoFlush` gives an assertion error.